### PR TITLE
feat: set impossible lock_transfer_block_id

### PR DESCRIPTION
**Pull request by zone117x**
@yknl 
> For the wallet issue that I mentioned during stand up, one way we can disable outgoing transactions from the wallet when we enter the freeze is to return a `lock_transfer_block_id` greater than the current block from the account status API endpoint on core:
https://core.blockstack.org/v1/accounts/1JSwmo6g8zgpKPMVRcrJBeH2oqJE8N9rax/STACKS/status
> 
> example return with `lock_transfer_block_id` set to 9999999:
   ```
address: "1JSwmo6g8zgpKPMVRcrJBeH2oqJE8N9rax"
block_id: 663762
credit_value: "1591022"
debit_value: "1270011"
lock_transfer_block_id: 9999999
txid: "e1d2702b865e180133690dd3dc022085168bf680f943853c53968572ed0a8d6c"
type: "STACKS"
vtxindex: 2263
   ```

### CODE REVIEW

**Review by yknl — APPROVED** [review]
Thanks @zone117x Looks good to me!

**Review by kantai — APPROVED** [review]

File: blockstack/blockstackd.py [hunk 1]
```diff
@@ -1423,7 +1423,7 @@ def export_account_state(self, account_state):
         """
         Make an account state presentable to external consumers
         """
```
Comment (zone117x) [inline blockstack/blockstackd.py:1425]: @jcnelson @kantai this location for a patch was not intuitive for me. There was another area that made more sense, but found this after some local testing. Please confirm

File: blockstack/blockstackd.py [hunk 2]
```diff
@@ -1433,6 +1433,11 @@ def export_account_state(self, account_state):
             'vtxindex': account_state['vtxindex'],
             'txid': account_state['txid'],
         }
+        # if block height is after the migration export threshold, return a lock height that will force the wallet to error when sending a tx
+        if result['block_id'] >= 665750:
```
Comment (diwakergupta) [inline blockstack/blockstackd.py:1437]: Can you use the constant IMPORT_BLOCK defined in Aaron's PR?

File: blockstack/blockstackd.py (hunk 1 above)
Comment (kantai) [reply blockstack/blockstackd.py:1425]: I think @jcnelson has the most context on the appropriate place to apply this. Alternatively, you can catch the forwarding that happens in `api/server.py` and rewrite the lock time there.

File: blockstack/blockstackd.py (hunk 2 above)
Comment (zone117x) [reply blockstack/blockstackd.py:1437]: idk how to import that from the other file. @kantai

Comment (diwakergupta) [reply blockstack/blockstackd.py:1437]: Not critical if that's not straight forward.

Comment (zone117x) [reply blockstack/blockstackd.py:1437]: Probably is but python module import is cancer

File: blockstack/blockstackd.py (hunk 1 above)
Comment (jcnelson) [reply blockstack/blockstackd.py:1425]: All this change will do is alter what is (a) returned by the RESTful API and (b) what is returned by `blockstack/lib/client.py`.  If that's what you want, then this is fine.  I think it's more appropriate to do it here than in `api/server.py`.

**Comment by jcnelson** [comment]
I'm a little confused as to what this is supposed to do.  Is it just supposed to trick the 1.0 wallet into thinking it can't spend STX because the tokens are all "locked"?

**Review by jcnelson — APPROVED** [review]

**Comment by kantai** [comment]
> I'm a little confused as to what this is supposed to do. Is it just supposed to trick the 1.0 wallet into thinking it can't spend STX because the tokens are all "locked"? If so, then the changes proposed here should be fine, but you'll want to of course test them with the 1.0 wallet.

Yep -- that's the idea.

I think this PR needs a slight tweak -- the `block_id` checked here isn't the current block height, but rather the `block_id` that the account was last modified at, correct?

**Comment by kantai** [comment]
Tested locally with an already passed block height, and this works correctly.

**Pull request merged**